app/assets/javascripts/discourse/app/controllers/edit-slow-mode.js

```diff
@@ -14,7 +14,7 @@ export default Controller.extend(ModalFunctionality, {
   minutes: null,
   seconds: null,
   saveDisabled: false,
-  enabledUntil: null,
+  enabledUntil: alias("model.slow_mode_enabled_until"),
```

Comment:
can we instead change the binding at the point of usage where it's written to? (It's probably in the .hbs file.)